scripts/check-go-vulnerabilities.sh

```diff
@@ -15,12 +15,30 @@ trap 'echo "Script terminated by signal" >&2; exit 143' TERM
 # TODO: Each ignored package should be removed from this list once the corresponding tracking
 #       ticket has been completed and the vulnerability has been addressed.
 IGNORED_PACKAGES=(
-    "docker"    # DEVPROD-25292 Docker vulnerability tracking
-    "stdlib"    # DEVPROD-25290 Stdlib vulnerability tracking
+    "docker"        # DEVPROD-25292 Docker vulnerability tracking
+    "stdlib"        # DEVPROD-25290 Stdlib vulnerability tracking
+    "rardecode"     #DEVPROD-25293 Archiver / Rardecode vulnerability tracking
+    "archiver/v3"   #DEVPROD-25293 Archiver / Rardecode vulnerability tracking
+    "csrf"          #DEVPROD-25429 CSRF vulnerability tracking
 )
 
-govul="govulncheck"
+# Validate that each ignored package has a tracking ticket
+script_content=$(cat "$0")
+ignored_section=$(echo "$script_content" | sed -n '/^IGNORED_PACKAGES=(/,/^)/p')
+package_lines=$(echo "$ignored_section" | grep '"' || true)
+
+missing_tickets=()
```

Comment:
This seems like a bit of overkill validation but I guess if the logic is already there to enforce it, that's also fine 👍 Next time I don't think you need to put too much time into enforcing that the script has tracking tickets, just making sure we can see a failure on the waterfall when there's new vulnerabilities would be sufficient for Evergreen devs to know they have to make tickets for new vulnerabilities.